_posts/0000-01-14-jayasaluja.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
 {: .center}
 
-Caption here
+Hello! This is Jaya!
```

Comment:
@jayasaluja , although I'm super happy to meet you, perhaps you should add a funny comment for the picture. 👍